frontend/src/components/SignForms/SignInForm/SignComponents/EmailField.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { TextField } from '@mui/material';
+import { EmailProps } from '../../SignInterfaces';
+import InputStyled from './styled';
+
+const EmailField = ({ login, touched, onChangeAccum, handleBlur, errors }: EmailProps) => (
+  <InputStyled
```

Comment:
I think here should be basic custom input from src/components later ( but it's not merged to develop yet )

Reply:
Let's leave it as it is. When 003 will drop I will replace it or fix it otherwise